tests/queries/0_stateless/02956_rocksdb_with_ttl.sql

```diff
@@ -0,0 +1,12 @@
+-- Tags: no-ordinary-database, use-rocksdb
+
+CREATE TABLE dict_with_ttl (key UInt64, value String) ENGINE = EmbeddedRocksDB(2) PRIMARY KEY (key);
```

Comment:
Should TTL to be present if definition?

Reply:
For `EmbeddedRocksDB` ttl is the first argument of the engine definition. I don't know why it was implemented this way in the first place, but it was.